up-and-runnig-with-recoil-a-new-state-management-library-for-react/01-use-recoil-atom-to-share-state-between-two-react-components.md

```diff
@@ -0,0 +1,41 @@
+# Notes
+
+# Set up environment 
+
+The documentation on how to set up the latest version of react can be found [here](https://reactjs.org/docs/create-a-new-react-app.html)
+
+1. Inside of an empty directory/folder run the following command to create a new react project `npx create-react-app running-w-recoil` 
+2. Run `cd running-w-recoil`
+3. Finally, to run your react server you could either run `npm start` or `yarn start`
+
+<TimeStamp start="0:00" end="0:02">
```

Comment:
this is a big ol' note and it's set to display for only 2 seconds

I think we want to generally think about how the notes will display to the user and they will popup and close based on the start/end times.

Reply:
I could leave this note to show up for 7 seconds, or just not include it in the timestamp as this was not explained by the instructor, it was more like an extra note for the learner. What do you think @joelhooks?